HOWTO-make-a-release.md

```diff
@@ -426,15 +426,21 @@ Then copy the result to the temporary directory on dev.openssl.org:
 To finish, log in on dev.openssl.org and send the signed Security
 Advisory by email as the user that signed the advisory.
 
-For all releases, send them to the default set of public mailing lists:
+For all releases, send it to the default set of public mailing lists:
 
     REPLYTO="openssl@openssl.org" mutt -s "OpenSSL Security Advisory" \
             openssl-project openssl-users openssl-announce \
+            </tmp/secadv_FILENAME.txt.asc
```

Comment:
Not sure why (@levitte might have an idea) when I was sending the advisory mail I had to use full e-mail addresses, i.e. openssl-project@openssl.org, otherwise the mail was not sent. Also I had to use EMAIL="tomas@openssl.org" otherwise tomas@dev.openssl.org would be used as the From address.

One option would be to use the `sudo -u openssl` however that causes messages (because they are gpg signed) to be displayed with errors like: `Valid signature, but sender address and signer address do not match (Tomáš Mráz <tomas@openssl.org>)` - this is what I see in Evolution when I open such message.

Reply:
I did some testing, and can see why the default configuration on dev.openssl.org doesn't agree with you.  The configuration isn't *wrong* per se, but it's at cross purpose with what you want to do.
 
I also recalled that I used to do this as the `openssl` user, not myself, which is something we used to do, quite a while ago now, and I've occasionally forgotten that.  As `openssl`, this works perfectly smoothly...  and the answer lies in `~openssl/.muttrc` and `~openssl/muttrc.aliases`, which you should have a look at.
So it seems that while we switched to running this command, the instructions essentially remained written for the `openssl` user.

Reply:
You could also simply copy `~matt/.muttrc`...  it's a bit simpler...

Reply:
My .muttrc is very simple!

````
$ cat .muttrc
set hostname = openssl.org
````

Should we add this to the instructions?

Reply:
I think it would be wiser to add a file `/etc/Muttrc.d/openssl.rc` with that setting, os it's set for everyone once and for all.

Reply:
Ok. In that case I assume that it is outside of the scope of this PR to handle this issue.

Reply:
Yes, of course.
I've now added `/etc/Muttrc.d/openssl-settings.rc`, it's a straight copy of `~matt/.muttrc`.  I tested it too, and it appears to do its job perfectly.

However, that still doesn't take away the need to set the EMAIL envvar to avoid having the mail sent as `{user}@dev.openssl.org`.  It's possible that mutt has a setting to fix that too...

Reply:
Hmm. I just use the command as stated in the instructions and don't have the EMAIL envvar set in my environment. It all seems to work ok for me.

Reply:
Ah yup, you're right, the test I did demonstrates that.  So ok, that resolves this matter then.